### PR TITLE
Make offer handling unit test not depend on database at compile time.

### PR DESCRIPTION
**Pull request by scrosby**
## Changes proposed in this PR

- Make offer handling unit test not depend on database at compile time.

## Why are we making these changes?
Helps with the postgres migration.

### CODE REVIEW

File: scheduler/test/cook/test/compute_cluster.clj [hunk 1]
```diff
@@ -51,6 +51,7 @@
     (is (= config (compute-cluster-config-ent->compute-cluster-config config-ent)))))
 
 (deftest test-db-config-ents
+  (setup)
```
Comment (nsinkov) [inline scheduler/test/cook/test/compute_cluster.clj:54]: how does this relate to the rest of the PR?

File: scheduler/test/cook/test/compute_cluster.clj [hunk 2]
```diff
@@ -192,6 +193,7 @@
            {:a {}}))))
 
 (deftest test-get-job-instance-ids-for-cluster-name
+  (setup)
```
Comment (nsinkov) [inline scheduler/test/cook/test/compute_cluster.clj:196]: how does this relate to the rest of the PR?

File: scheduler/test/cook/test/compute_cluster.clj (hunk 1 above)
Comment (scrosby) [reply scheduler/test/cook/test/compute_cluster.clj:54]: Unit tests need to run this to setup mount state, however, a lot omit it and piggyback on prior tests having had already run mount. I encountered this test breaking and fixed it.

File: scheduler/test/cook/test/compute_cluster.clj (hunk 2 above)
Comment (scrosby) [reply scheduler/test/cook/test/compute_cluster.clj:196]: Unit tests need to run this to setup mount state, however, a lot omit it and piggyback on prior tests having had already run mount. I encountered this test breaking and fixed it.

**Pull request merged**